README.md

```diff
@@ -138,12 +145,109 @@ make test-ts-binary-rococo
 
 Remember to run the clean-up afterwards.
 
-## extend the leasing period
+## License
 
-The default leasing duration for parachain is 1 day, in case you want to extend it (even after it's downgraded to parathread), simply do a `forceLease` via sudo, it should be upgraded to parachain soon again and start to produce blocks.
+GPLv3
 
-![image](https://user-images.githubusercontent.com/7630809/135689832-1f57cd5c-7f83-4fce-9bb0-832b77a38dcc.png)
 
-## License
+## How to Build and Run Parachain and Tee-worker
 
-GPLv3
+## Preparation
+
+1. env: [Setup **SGX TEE** Environment](https://web3builders.notion.site/Setup-SGX-TEE-Environment-68066770831b45b7b632e682cf159477?pvs=4) 
+2. add VSCode configs for indexing and parsing tee-worker codes
+
+```
+"rust-analyzer.linkedProjects": [
+        "tee-worker/Cargo.toml"
+    ],
+```
+
+## Build
+
+We need to build the `integritee-service` before we launch the entire system
+
+```
+git clone https://github.com/litentry/litentry-parachain
+cd ./litentry-parachain/tee-worker
+source /opt/intel/sgxsdk/environment
+SGX_MODE=SW make
+```
+
+## Launch
+
+### 1. Start a local docker setup
+
+In order to create a local docker setup, you can run the following command
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json
+```
+This will create three docker containers, 2 Relay Chain Validators and 1 Parachain Collator. However it will be using the default ports as present in .env.dev . If you want to run the system by offseting the default ports by a certain value. You can run this command instead:
+
+```
+./local-setup/launch.py local-setup/development-worker.json local-docker 100
+```
+This will run the same containers and use the offset value of 100.
+
+### 2. Start a local binary setup
+
+In order to create a local binary setup, using default ports, you can run the following command:
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json local-binary
+```
+
+If you want to launch the same system by offseting the port values, you can use this command 
+```
+./local-setup/launch.py local-setup/development-worker.json local-binary 100
```

Comment:
same config here

Reply:
One is for local docker and the other for local binary

Reply:
One is for local docker and the other for local binary

Reply:
I think you misunderstood. Either `local docker` or `local binary`, that refers to parachain. Config file here is for used for worker.